.github/workflows/ci.yml

```diff
@@ -15,3 +15,49 @@ jobs:
     - uses: olafurpg/setup-scala@v5
     - name: Tests
       run: sbt -J-Xms4G -J-Xmx4G test
```

Comment:
Hi @gvolpe I believe the error is due to the old definition being here. You can only have one definition of `jobs` / `on`

Reply:
you can probably save the new one to a new workflow if you want to test both

Reply:
Damn yeah, I appended the content of the `dhall-to-yaml` to the existing file instead of override it :man_facepalming: 

Still, doesn't seem to be picked up. The only difference with the original example is this:

```yaml
on:
  pull_request: {}
  push:
    branches:
      - master
```

Anything wrong with it? Thanks for looking into it!

Reply:
Nevermind, got it! Had to modify the `needs` value since I removed the `checks` step... Thanks again for helping me out, this is awesome! :)